### PR TITLE
[CircleGraph] Introduce CircleGraphCtrl

### PR DESCRIPTION
**Pull request by seanshpark**
This will extract part of CircleGraphPanel to as CircleGraphCtrl to
provide common CircleGraph as a UI component.
- this change will extract getHtmlForWebview and related

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #641, from draft #647

File: src/CircleGraph/CircleGraphCtrl.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+};
+
+function getNonce() {
+  let text = '';
+  const possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  for (let i = 0; i < 32; i++) {
+    text += possible.charAt(Math.floor(Math.random() * possible.length));
+  }
+  return text;
+}
```
Comment (jyoungyun) [inline src/CircleGraph/CircleGraphCtrl.ts:95]: This code will be replaced by #679, right?

Comment (seanshpark) [reply src/CircleGraph/CircleGraphCtrl.ts:95]: Yup :)

Comment (jyoungyun) [reply src/CircleGraph/CircleGraphCtrl.ts:95]: @seanshpark Do you want to merge it first? :)

Comment (seanshpark) [reply src/CircleGraph/CircleGraphCtrl.ts:95]: this and #679 does not depend on each other. it's ok with the order.

**Review by hyunsik-yoon — APPROVED** [review]

**Pull request merged**